config/environments/development.rb

```diff
@@ -54,3 +54,34 @@
   # Allow requests for all domains e.g. <app>.dev.gov.uk
   config.hosts.clear
 end
+
+# make discovery work over HTTP
+module OpenIDConnect
+  module Discovery
+    module Provider
+      class Config
+        class Resource
+          def initialize(uri)
```

Comment:
So i get this has to be called explcitly. And that a sensible dev should only ever do that in our dev environmnet...

But I'm wondering if it's extra paranoid to add some sort of dump out check here... Like if it's initialized in the wrong environment then raise an error instead of executing?

Take it or leave it!

Reply:
Since this is in the `development.rb` config file, I can't see any way this would be executed outside of development?

Reply:
cool cool